private/pkg/app/appflag/builder.go

```diff
@@ -63,13 +64,8 @@ func newBuilder(appName string, options ...BuilderOption) *builder {
 }
 
 func (b *builder) BindRoot(flagSet *pflag.FlagSet) {
-	// True means debug, false means info.
-	flagSet.BoolVarP(&b.verbose, "verbose", "v", false, "Enable verbose logging.")
-	flagSet.StringVar(&b.logLevel, "log-level", "", "The log level [debug,info,warn,error].")
-	// Effectively deprecated - we only ever cared about debug vs info, which is now controlled
-	// by verbose, but we keep this around in case it is being used and handle it being set.
-	// Note that both verbose and log-level cannot be set together.
-	_ = flagSet.MarkHidden("log-level")
+	flagSet.BoolVarP(&b.verbose, "verbose", "v", false, "Turn on verbose mode.")
+	flagSet.BoolVar(&b.debug, "debug", false, "Turn on debug logging.")
```

Comment:
How do these interact with each other? Does `debug` imply `verbose`? Worth adding to the help I think.

Reply:
They don't, they're completely separate